src/Style/Sass/pages/_resetPassword.scss

```diff
@@ -0,0 +1,98 @@
+.resetPassword {
+  position: relative;
+  margin: 25% 0 0 32%;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
put the color in the ``_variables `` for all use and drop the name on the channel to reduce duplication

Reply:
Noted @truestbyheart